app/Http/Controllers/Api/Instructor/AttendanceController.php

```diff
@@ -87,12 +88,11 @@ public function store(StoreRequest $request): JsonResponse
      */
     public function show(ShowRequest $request): AttendanceShowResource
     {
-        $instructorId = Auth::guard('instructor')->user()->id;
+        $instructor = Auth::guard('instructor')->user();
         $courseId = $request->course_id;
         $course = Course::with('tags')->findOrFail($courseId);
 
-        if ($course->instructor_id !== $instructorId) {
-            // ログインしている講師の講座でない場合はエラーを返す
+        if (! (new AttendancePolicy)->view($instructor, $course)) {
             throw new AuthorizationException('Forbidden, invalid instructor_id.');
```

Comment:
次のようにクラスとポリシーのマッピングを明示的に指定できるようになりますので、修正をお願いします！
```suggestion
$this->authorize('view', [Attendance::class, $course]);
```